src/test/resources/docker-db.yml

```
@@ -21,13 +21,13 @@ services:
       MYSQL_ROOT_PASSWORD: root
     hostname: mysql
   clickhouse:
-    image: yandex/clickhouse-server
+    image: clickhouse/clickhouse-server:21.10
```

We use this version in other projects already
```suggestion
    image: clickhouse/clickhouse-server:22.5-alpine
```